### PR TITLE
[changelog-parser] Add missing option parameter (text); add interface for result type

### PR DESCRIPTION
**Pull request by osteele**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/hypermodules/changelog-parser#readme
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/changelog-parser/index.d.ts [hunk 1]
```diff
@@ -1,25 +1,68 @@
...
+     */
+    text: string;
     /**
      * Removes the markdown markup from the changelog entries by default.
      * You can change its value to false to keep the markdown.
      */
     removeMarkdown: boolean;
 }
 
+// This type can't be exported, because the module interface has a default
```
Comment (gabritto) [inline types/changelog-parser/index.d.ts:38]: so, if you want to export the `Changelog` interface, you can use (declaration merging)[https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-namespaces] to have a namespace declaring the types, with the same name of the function, like so:
```ts
declare namespace parseChangelog {
    interface Options { ... }
    interface Changelog { ... }
}
declare function parseChangelog();
export = parseChangelog;
```
then you can refer to the types when importing from another file, like so:
```ts
import changelog from "parseChangelog";
changelog(...);
var opt: changelog.Options;
var log: changelog.Changelog;
```

**Review by gabritto — CHANGES_REQUESTED** [review]
I think this looks good, but it seems you want to also export the interface types declared here, so I left a comment on how you can do this, and then you can also update the tests accordingly.
Let me know if there's something you don't understand and I'm happy to help.

File: types/changelog-parser/index.d.ts (hunk 1 above)
Comment (osteele) [reply types/changelog-parser/index.d.ts]: Thank you, I have revised the PR to use your recommendation.

**Review by gabritto — APPROVED** [review]
thanks :)

**Pull request merged**